servant-routes-golden/servant-routes-golden.cabal

```diff
@@ -0,0 +1,98 @@
+cabal-version:      3.0
+name:               servant-routes-golden
+version:            0.1.0.0
+synopsis:           Golden test your Servant APIs using `servant-routes`
+description:
+    See the documentation of 'Servant.API.Routes.Golden'.
+license:            BSD-3-Clause
+license-file:       LICENSE
+author:             Frederick Pringle
+maintainer:         freddyjepringle@gmail.com
+copyright:          Copyright(c) Frederick Pringle 2025
+homepage:           https://github.com/fpringle/servant-routes/servant-routes-golden
+bug-reports:        https://github.com/fpringle/servant-routes/issues
+category:           Servant, Web
+build-type:         Simple
+extra-doc-files:    CHANGELOG.md
+                    README.md
+
+tested-with:
+    GHC == 8.8.4
+  , GHC == 8.10.7
+  , GHC == 9.0.2
+  , GHC == 9.2.4
+  , GHC == 9.2.8
+  , GHC == 9.4.2
+  , GHC == 9.4.5
+  , GHC == 9.6.1
+  , GHC == 9.6.7
+  , GHC == 9.8.2
+  , GHC == 9.10.2
+  , GHC == 9.12.2
+
+source-repository head
+  type:           git
+  location:       https://github.com/fpringle/servant-routes
+
+common warnings
+  ghc-options: -Wall -Wno-unused-do-bind
+
+common deps
+  build-depends:
+    , base >= 4.13 && < 5
+    , aeson >= 2.0 && < 2.3
+    , text >= 1.2 && < 2.2
+    , aeson-pretty >= 0.8.8 && < 0.9
+    , hspec-core >= 2.10.0 && < 2.12
+    , hspec-golden >= 0.2 && < 0.3
+    , servant-routes >= 0.1.0.0 && < 0.2
+
+common extensions
+  default-extensions:
+    DeriveGeneric
+    DerivingVia 
+    FlexibleContexts
+    FlexibleInstances
+    LambdaCase
+    NamedFieldPuns
+    OverloadedStrings
+    PackageImports
+    RecordWildCards
+    ScopedTypeVariables
+    TypeApplications
+    ViewPatterns
+    DataKinds
+    AllowAmbiguousTypes
+    TypeFamilies
+    TypeOperators
+
+library
+  import:
+      warnings
+    , deps
+    , extensions
+  ghc-options: -Wunused-packages
+  exposed-modules:
+    Servant.API.Routes.Golden
+  hs-source-dirs:   src
+  default-language: Haskell2010
+
+test-suite servant-routes-golden-test
+  import:
+      warnings
+    , deps
+    , extensions
+  default-language: Haskell2010
+  type:             exitcode-stdio-1.0
+  hs-source-dirs:   test
+  main-is:          Spec.hs
+  other-modules:
+    Servant.API.Routes.GoldenSpec
+  build-tool-depends:
+      hspec-discover:hspec-discover
+  ghc-options:      -Wno-orphans
+  build-depends:
+    , servant-routes-golden
+    , servant >= 0.17 && < 0.21
+    , hspec >= 2.9 && < 2.12
+    , QuickCheck >= 2.14 && < 2.15
```

Comment:
newest QuickCheck is 2.15.0.1, it would be nice to accept that. `cabal outdated` or Renovate can help you find outdated bounds